agent_build_refactored/tools/runner.py

```diff
@@ -988,10 +992,18 @@ def get_remote_docker_host_for_step(step: RunnerStep) -> Optional[str]:
                 return None
 
             # Get EC2 AMI image according to step's architecture.
+
             ec2_image = DOCKER_EC2_BUILDERS.get(step.architecture)
             # If image is not found then just run in local docker engine.
-            if ec2_image is None:
-                return None
+            if ec2_image is None and step.architecture != Architecture.X86_64:
+                ec2_image = EC2DistroImage(
+                    image_id="ami-09d56f8956ab235b3",
```

Comment:
Should probably make it a module level constant with some comment on what image it is, etc.

Reply:
It's also a leftover from my attempts to compile Python for other platform, removed it for now.